myst_nbs/variational_inference/pathfinder.myst.md

```diff
@@ -0,0 +1,82 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.7
+kernelspec:
+  display_name: pymc4
+  language: python
+  name: pymc4
+---
+
+(pathfinder)=
+
+# Pathfinder Variational Inference
+
+:::{post} Sept 30, 2022 
+:tags: variational inference, jax 
+:category: advanced 
```

Comment:
I would use the two categories here, `advanced` for the level and `how-to` for the type of content